src/cryptoadvance/specter/specter.py

```diff
@@ -428,7 +441,11 @@ def update_alt_rate(self, alt_rate, user):
     def update_alt_symbol(self, alt_symbol, user):
         self.config_manager.update_alt_symbol(alt_symbol, user)
 
-    # mark logic!
+    def update_services(self, services):
+        """takes a list of service_names which should be activated"""
+        self.config["services"] = services
+        self._save()
+
```

Comment:
This is no longer needed as with this commit, services are dependent on the user:
https://github.com/k9ert/specter-desktop/commit/6d64a2964f9a7cc3d0b36e5b0d455fc7b6569430#diff-3a0c72fdfb0c27876d7c5efc052274c635e69d022508f8bf2f93c56ceaf7a9de
(However this code did not get deleted)